.github/workflows/nx-affected-e2e.yml

```diff
@@ -22,6 +22,10 @@ jobs:
             - name: Install Playwright Browsers
               shell: bash
               run: yarn playwright install --with-deps
+            - name: start cloud server worker
```

Comment:
is this needed since there's a npm script that does this already?

Reply:
that script is not getting called and I was getting a fetch failed error so explicitly starting seems to be working